docs/src/rules/camelcase.md

```diff
@@ -319,7 +319,7 @@ function UNSAFE_componentWillMount() {
 
 :::
 
-::: correct
+::: correct { "sourceType": "script" }
```

Comment:
I think a better fix here would be to change the name of the second function so that it's not a duplicate but also matches the `^UNSAFE_` pattern. It seems very unlikely that the intent was really to redeclare the function. It's much more likely that the intent was to show that the regex can allow multiple different names.